Schema Statements.sql

```diff
@@ -0,0 +1,135 @@
+CREATE DATABASE  IF NOT EXISTS `new_schema` /*!40100 DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci */ /*!80016 DEFAULT ENCRYPTION='N' */;
+USE `new_schema`;
+-- MySQL dump 10.13  Distrib 8.0.25, for Win64 (x86_64)
```

Comment:
Remove all the export comments, makes it hard to read.
We will probably need to tweak this by hand anyway.